tutorials/migrating/upgrading_to_godot_4.rst

```diff
@@ -438,6 +442,30 @@ table to find its new name.
 - TileMap's ``world_to_map()`` is now ``local_to_map()``.
 - TileMap's ``map_to_world()`` is now ``map_to_local()``.
 - Transform2D's ``xform()`` is ``mat * vec`` and ``xform_inv()`` is ``vec * mat``.
+- XRPositionalTracker's ``_set_name()`` is now ``get_tracker_name()``.
+- BaseButton's ``_unhandled_input()`` is now ``_unhandled_key_input()``.
+- ViewportContainer's ``_unhandled_input()`` is now ``_unhandled_key_input()``.
+- AnimationPlayer's ``add_animation()`` is now ``add_animation_library()``.
```

Comment:
Needs more explanation as it's not a direct replacement

Reply:
In 4.2 this was also moved to class AnimationMixer from AnimationPlayer, so which option is preferable?

1. AnimationPlayer's ``add_animation()`` is now ``add_animation_library()`` and now uses :ref:`class_AnimationLibraries`.
2. AnimationPlayer's ``add_animation()`` was moved to :ref:`class_AnimationMixer`'s ``add_animation_library()`` animations will need to be in a :ref:`class_AnimationLibraries`.
3. AnimationPlayer's ``add_animation()`` is now ``add_animation_library()`` and now uses :ref:`class_AnimationLibraries`. (The method was moved to :ref:`class_AnimationMixer` in Godot 4.2)

Reply:
I think 1 is good